libc-bottom-half/headers/public/wasi/preview2.h

```diff
@@ -0,0 +1,2481 @@
+// Generated by `wit-bindgen` 0.16.0. DO NOT EDIT!
+#ifndef __BINDINGS_IMPORTS_H
+#define __BINDINGS_IMPORTS_H
```

Comment:
Maybe it would be good if the tool could be passed the name `preview2` so that these lines would make sense, and then we could also drop the `sed` and rename logic in the Makefile?

Reply:
Agreed; I'll look into that.

Reply:
https://github.com/bytecodealliance/wit-bindgen/pull/805